src/scripts/fetch_package_names.js

```diff
@@ -43,6 +78,11 @@ function fetchPackageNames(options, cb) {
       type: 'token',
       token: process.env.GITHUB_TOKEN
     });
+
+    github.misc.getRateLimit({}, (err, res) => {
```

Comment:
What's the goal of this besides probably debugging on CI?

Reply:
That is the goal